include/sprite.h

```diff
@@ -84,7 +84,7 @@ struct UNK_0808B3FC_UNK240 {
     /* 0x0C */ struct UNK_0808B3FC_UNK240_UNKC *unkC;
     /* 0x10 */ u32 unk10; // bitfield
 
-    /* 0x14 */ u16 unk14;
+    /* 0x14 */ u16 unk14; // animation cursor
```

Comment:
I was actually thinking this could be named something to do with "instruction pointer"

Reply:
Leave as is I guess